### PR TITLE
code: add page

### PR DESCRIPTION
**Pull request by gldraphael**
This PR adds a tldr page for [VS Code](http://code.visualstudio.com/).

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/code.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# code
+
+> Visual Studio Code   
+> Code editing. Redefined.
```
Comment (agnivade) [inline pages/common/code.md]: This line seems to be more subjective and less informative. Is it absolutely needed ?

File: pages/common/code.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Open VS Code
+
+`code`
+
+- Open current diretory in VS Code
```
Comment (agnivade) [inline pages/common/code.md]: diretory => directory

File: pages/common/code.md (hunk 1 above)
Comment (gldraphael) [reply pages/common/code.md]: Probably not. I picked it up from their website. I'll remove that.

**Comment by agnivade** [comment]
Strange, the travis build is not triggering.

File: pages/common/code.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# code
+
+> Visual Studio Code.
+
+- Open VS Code:
+
+`code`
+
+- Open current directory in VS Code:
```
Comment (sbrl) [inline pages/common/code.md]: Perhaps `Open the current directory in VS Code:` would read better?

File: pages/common/code.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+`code .`
+
+- Open file or directory in VS Code:
```
Comment (sbrl) [inline pages/common/code.md]: A similar thing here, too. Perhaps `Open a file` would read better?

File: pages/common/code.md (hunk 3 above)
Comment (gldraphael) [reply pages/common/code.md]: Oh yeah. I've almost stopped using English articles ever since I started to make my commit messages shorter 😅

**Comment by sbrl** [comment]
Looks ok to me! I'm not sure what's up with travis, though. Shall I merge anyway, @agnivade?

**Comment by agnivade** [comment]
Yea, no idea whats up with travis.

**Pull request merged**